wishlist.graphql

```diff
@@ -152,12 +202,30 @@ type Wishlist {
 	Timestamp when Wishlist was last updated.
 	"""
 	lastUpdatedAt: DateTime!
+	internalProductVariants: [ProductVariant!]!
```

Comment:
That's a good question, it is flagged as invisible but apparently still lands in the schema.

Reply:
The problem is, this will lead to issues with the federation support as the gateway uses this static schema, but then it is no longer there at runtime
Any idea how we can get rid of this?

Reply:
Fixed it with another flag: `visible = false` -> `skip`